src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1427,6 +1428,19 @@ public void fillMsqOptionForNewQuestion(int optionIndex, String optionText) {
         fillTextBox(optionBox, optionText);
     }
 
+    public void dragAndDropQuestionOption(String qnType, int qnNumber, int sourceIndex, int targetIndex) throws Exception {
+        WebElement draggedOptionElement = browser.driver.findElement(By.xpath("//div[@id='" + qnType + "OptionRow-"
+                + sourceIndex + "-" + qnNumber + "']//span[@class='glyphicon glyphicon-resize-vertical']"));
+        WebElement targetElement = browser.driver.findElement(By.xpath("//div[@id='" + qnType + "OptionRow-"
+                + targetIndex + "-" + qnNumber + "']//span[@class='glyphicon glyphicon-resize-vertical']"));
+
+        Actions builder = new Actions(browser.driver);
+        builder.clickAndHold(draggedOptionElement).perform();
+        builder.moveToElement(targetElement, 0, 10).perform(); // drag option 10 units below target
+        builder.moveToElement(targetElement).perform(); // drag option to target
+        builder.moveToElement(targetElement, 0, -10).release().perform(); // drag option 10 units above target and release
```

Comment:
I understand from your other comment that just using `dragAndDrop` does not work, thus you have to break it down into smaller movements like this. However, I'm not convinced that this is the best solution. Have you tried moving straight away to `(targetElement, 0, -10)`, and then wait for a few miliseconds before releasing?

Reply:
i think i tried it using `clickAndHold` at the time but it wasn't working. Let me give this a shot again

Reply:
@tran-tien-dat  ok so I researched this again and realized that I was forgetting to use the `build` method before using `perform` which is necessary for an action chain. Not the simulation works without adding any delay